amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/disposition/DispositionServiceImpl.java

```diff
@@ -473,16 +473,13 @@ public NodeRef getAssociatedRecordsManagementContainer(DispositionSchedule dispo
             List<ChildAssociationRef> assocs = this.nodeService.getParentAssocs(dsNodeRef, ASSOC_DISPOSITION_SCHEDULE, RegexQNamePattern.MATCH_ALL);
             if (!assocs.isEmpty())
             {
-                if (assocs.size() != 1)
```

Comment:
Even better than moving this log level check would be to remove it by relying on slf4j instead:
```
LOGGER.atWarn().log("Something something... {}", dispositionSchedule.getNodeRef());
```

Reply:
done the changes as per the comment.